test/parallel/test-net-listen-anyaddr.js

```diff
@@ -0,0 +1,10 @@
+var assert = require('assert');
+var cp = require('child_process');
+
+var args = [
+  '-e',
+  "require('net').createServer().listen(0, function() { this.close(); });",
+];
+
+// should exit cleanly almost immediately
+cp.execFile(process.execPath, args, {timeout: 500}, assert.ifError);
```

Comment:
Not to be that guy but was the `500` timeout randomly picked? We want each test to exit as fast as possible so if this can be reduced, that'd be great - if not, thats okay too, thoughts?